rmf_demos_tasks/rmf_demos_tasks/emergency_signal.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+# Copyright 2025 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Publish an emergency signal."""
+
+import argparse
+import sys
+
+import rclpy
+from rclpy.node import Node
+from rclpy.qos import QoSDurabilityPolicy as Durability
+from rclpy.qos import QoSHistoryPolicy as History
+from rclpy.qos import QoSProfile
+from rclpy.qos import QoSReliabilityPolicy as Reliability
+
+from rmf_fleet_msgs.msg import EmergencySignal
+
+###############################################################################
+
+def str2bool(v):
```

Comment:
Style test is asking for two blank lines before this def. Alternatively could probably just delete line 30.